conda-recipe/run_test.bat

```diff
@@ -15,34 +15,18 @@ rem See the License for the specific language governing permissions and
 rem limitations under the License.
 rem ============================================================================
 
-rem %1 - scikit-learn-intelex repo root
+rem %1 - scikit-learn-intelex repo root (leave empty if it's PWD)
 
-set MPIROOT=%PREFIX%\Library
 set exitcode=0
 
-IF DEFINED DPCPPROOT (
-    echo "Sourcing DPCPPROOT"
-    call "%DPCPPROOT%\env\vars.bat" || set exitcode=1
-    set "CC=dpcpp"
-    set "CXX=dpcpp"
-    dpcpp --version
-)
+IF NOT DEFINED PYTHON (set PYTHON="python")
 
-IF DEFINED DALROOT (
-    echo "Sourcing DALROOT"
-    call "%DALROOT%\env\vars.bat" || set exitcode=1
-    echo "Finish sourcing DALROOT"
-)
+%PYTHON% -c "from sklearnex import patch_sklearn; patch_sklearn()" || set exitcode=1
 
-IF DEFINED TBBROOT (
-    echo "Sourcing TBBROOT"
-    call "%TBBROOT%\env\vars.bat" || set exitcode=1
-)
+%PYTHON% -m unittest discover -v -s %1%tests -p test*.py || set exitcode=1
 
-%PYTHON% -m unittest discover -v -s %1\tests -p test*.py || set exitcode=1
-
-pytest --verbose --pyargs %1\daal4py\sklearn || set exitcode=1
+pytest --verbose --pyargs daal4py || set exitcode=1
 pytest --verbose --pyargs sklearnex || set exitcode=1
-pytest --verbose --pyargs %1\onedal --deselect="onedal/common/tests/test_policy.py" || set exitcode=1
```

Comment:
Does private CI use run_test.bat? I know that the public windows doesn't build the dpc portion and test against dpctl/dpnp so some of these tests aren't verified against public CI.  May be worth running an /intelci: run and checking the testing. It looks like it was deselected in your #1115 but there isn't a comment as to why.

Reply:
Private CI doesn't use run_test.bat, and I don't remember reason why deselection was introduced. Anyway, it works now.